### PR TITLE
[IMP] purchase_order_product_recommendation: show all products

### PR DESCRIPTION
**Pull request by chienandalu**
- Add flag to be able to load all the products with supplier infos for
ther current customer

cc @Tecnativa

### CODE REVIEW

File: purchase_order_product_recommendation/wizards/purchase_order_recommendation.py [hunk 1]
```diff
@@ -36,11 +36,16 @@ class PurchaseOrderRecommendation(models.TransientModel):
...
         string='Number of recommendations',
-        default=0,
+        default=15,
         required=True,
         help='Stablish a limit on how many recommendations you want to get.'
              'Leave it as 0 to set no limit',
     )
+    show_all_partner_products = fields.Boolean(
+        string='Show all products',
+        default=False,
```
Comment (nikul-serpentcs) [inline purchase_order_product_recommendation/wizards/purchase_order_recommendation.py:46]: @chienandalu no need to `default=False`

Comment (pedrobaeza) [reply purchase_order_product_recommendation/wizards/purchase_order_recommendation.py:46]: @nikul-serpentcs remember about "explicit better than implicit", but any way, I think you should better focus your reviews about explicit guidelines or potential problems in the code, the UI, etc, as these nitpickings are not adding value to the review.

**Review by pedrobaeza — COMMENTED** [review]
Please increase code coverage

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@nikul-serpentcs please re-review

**Review by nikul-serpentcs — APPROVED** [review]
Code Review LGTM :+1: 

Go Ahead @pedrobaeza

**Pull request merged**